src/packagedcode/pubspec.py

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright (c) 2019 nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/scancode-toolkit/
+# The ScanCode software is licensed under the Apache License version 2.0.
+# Data generated with ScanCode require an acknowledgment.
+# ScanCode is a trademark of nexB Inc.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with ScanCode or any ScanCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  ScanCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  ScanCode is a free software code scanning tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import io
+import logging
+import sys
+
+import yaml
+import attr
+from packageurl import PackageUrl
+from commoncode import filetype
+from packagedcode import models
+
+from packagedcode import models
+
+TRACE = False
+
+def logger_debug(*args):
+    pass
+
+logger = logginf.getLogger(__name__)
+
+if TRACE:
+    logging.basicConfig(stream = sys.stdout)
+    logger.setLevel(logging.DEBUG)
+    
+    def logger_debug(*args):
+        return logger.debug(''.join(isinstance(a, string_types) and a or repr(a) for a in args))
+
+@attr.s()
+class PubspecPackage(models.Package):
+    metafiles = ('pubspec.yaml',)
+    extensions = ('.yaml',)
+    default_type = 'pubspec'
+    default_primary_language = 'dart'
+    default_web_baseurl = 'https://dart.dev/tools/pub/pubspec
+    default_download_baseurl = 'https://dart.dev/tools/pub/pubspec'
+    default_api_baseurl = 'https://dart.dev/tools/pub/pubspec'
+
+    @classmethod
+    def recognize(cls, location):
+        yield parse(location)
+
+    def repository_homepage_url(self, baseurl=default_web_baseurl):
+        return '{}/pods/{}'.format(baseurl, self.name)

```

Comment:
I think you have used Cocoapods link here. Update this with dart package.

Reply:
> 
> 
> I think you have used Cocoapods link here. Update this with dart package.

Ok, I'll do that.